tensorflow_addons/custom_ops/image/cc/kernels/euclidean_distance_transform_op_gpu.cu.cc

```diff
@@ -20,15 +20,48 @@ limitations under the License.
 #include "tensorflow/core/framework/register_types.h"
 #include "tensorflow/core/framework/types.h"
 #include "tensorflow/core/platform/types.h"
+#include "tensorflow/core/util/gpu_device_functions.h"
+#include "tensorflow/core/util/gpu_kernel_helper.h"
 #include "tensorflow_addons/custom_ops/image/cc/kernels/euclidean_distance_transform_op.h"
 
 namespace tensorflow {
 namespace addons {
 
 namespace functor {
 
-// Explicit instantiation of the GPU functor.
-typedef Eigen::GpuDevice GPUDevice;
+template <typename T>
+__global__ void EuclideanDistanceTransformGPUKernel(
+    const int batch_size, const T *input_ptr, const int input_height,
+    const int input_width, const int input_channel, T *output_ptr,
+    const int output_height, const int output_width, const int output_channel) {
+  typename TTypes<T, 4>::ConstTensor images(input_ptr, batch_size, input_height,
+                                            input_width, input_channel);
+  typename TTypes<T, 4>::Tensor output(output_ptr, batch_size, output_height,
+                                       output_width, output_channel);
+  auto edt_generator =
+      EuclideanDistanceTransformGenerator<GPUDevice, T>(images);
+  for (int k : GpuGridRangeX<int>(images.dimension(0))) {
+    edt_generator(output, k);
+  }
+}
+
+template <typename T>
+struct EuclideanDistanceTransformFunctor<GPUDevice, T> {
+  typedef typename TTypes<T, 4>::ConstTensor InputType;
+  typedef typename TTypes<T, 4>::Tensor OutputType;
+
+  void operator()(OpKernelContext *ctx, OutputType *output,
+                  const InputType &images) const {
+    auto d = ctx->eigen_device<GPUDevice>();
+    GpuLaunchConfig config = GetGpuLaunchConfig(images.size(), d);
```

Comment:
@fsx950223 
Maybe
```python
    GpuLaunchConfig config = GetGpuLaunchConfig(images.dimension(0), d);
```

Reply:
Fix the memory issue but crashed.

Reply:
Do you have other logs?

Reply:
Maybe run on colab with GDB.

Reply:
Here https://github.com/tensorflow/tensorflow/blob/dec8e0b11f4f87693b67e125e67dfbc68d26c205/tensorflow/core/kernels/dilation_ops_gpu.cu.cc#L196

Reply:
Hmm, it's weird.